### DISCUSSION TITLE
Daily 12/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#125] [General]
Ayer terminé el análisis completo, faltaría pasar todo a "documentación" formal en PDF. Hoy a la mañana hicimos una reunión para explicar algo del análisis y ver algo del diseño también, a la noche tendría que empezar con el diseño para tener como mínimo uno de los diseños terminado.

**Comment by NataliaAlvarezIspc** [comment]
Ayer subí todo todos los html y agregué los estilos a css. También estuve avanzando un poco con Python.
Hoy participé en la reunión  virtual de análisis y diseño. Estaré haciendo el index.html y corrigiendo algunos estilos de lo hecho.

**Comment by AhumadaCeleste** [comment]
Ayer rendi modulo de python y participe de la reunión con perdidas de contenido por el corte en el audio de conexión, y por estar en el ámbito laboral. Hoy voy a practicar python y a tratar de terminar con css. Mañana espero poder hacer lo de boobstrap

**Comment by Brian-Almada** [comment]
Ayer pude ver algo de boostrap y el video de la reunión. Hoy voy a hacer las actividades de ingles que no pude hacer ayer, ver los nuevos videos de las reuniones y un poco más de boostrap.
